corehq/apps/app_manager/models.py

```diff
@@ -5560,7 +5558,7 @@ def validate_app(self):
             errors.extend(module.validate_for_build())
 
         for form in self.get_forms():
-            errors.extend(form.validate_for_build(validate_module=False))
+            errors.extend(form.validate_for_build())
```

Comment:
Looking at this again and where it's called I think this was added to prevent duplicate work. This is the only place this is called with `validate_module=False` and it's inside the `app.validate_app()` function which already does module validation. 

It's possible that the logic in `form.validate_for_build` and `module.validate_for_build` have somehow diverged so that calling `form.validate_for_build(validate_module=True)` does more module level validation than `module.validate_for_build()` but if that's the case then it should probably be fixed to be consistent.

I don't have any other concerns about this change other than the performance implications of doing the module level validation repeatedly for every form in the app.